examples/serverless-raw/index.ts

```diff
@@ -140,5 +140,10 @@ let integration = new aws.apigateway.Integration("myrestapi-integration", {
 let deployment = new aws.apigateway.Deployment("myrestapi-deployment-prod", {
     restApi: restApi,
     description: "my deployment",
-    stageName: "prod",
```

Comment:
stageName is deprecated and now requires you to use the separate resource. Will need to add to upgrade guide.